build_runner/test/asset_graph/graph_test.dart

```diff
@@ -117,18 +117,10 @@ void main() {
           }
         }
 
-        var encoded = JSON.encode(graph.serialize());
-        var decoded = new AssetGraph.deserialize(JSON.decode(encoded) as Map);
+        var encoded = graph.serialize();
+        var decoded = new AssetGraph.deserialize(encoded);
         expect(graph, equalsAssetGraph(decoded));
       });
-
-      test('Throws an AssetGraphVersionError if versions dont match up', () {
```

Comment:
We should restore this test in some manner

Reply:
Done